src/renderer/frames/cluster-frame/init-cluster-frame/init-cluster-frame.injectable.ts

```diff
@@ -12,16 +12,17 @@ import appEventBusInjectable from "../../../../common/app-event-bus/app-event-bu
 import clusterFrameContextInjectable from "../../../cluster-frame-context/cluster-frame-context.injectable";
 
 const initClusterFrameInjectable = getInjectable({
-  instantiate: (di) =>
-    initClusterFrame({
+  instantiate: (di) => {
+    return initClusterFrame({
```

Comment:
I don't think you need to touch this in this PR

Reply:
Yes, forgot to change this back. Reverting change.